translations/messages.fr.xlf

```diff
@@ -0,0 +1,3070 @@
+        <source>A confirmation email will be sent to validate your account</source>
+        <target>Un email de confirmation vous sera envoyé pour valider votre compte</target>
+      </trans-unit>
+      <trans-unit id="77522255637065336" datatype="html">
+        <source>Connection error</source>
+        <target>Erreur de connexion</target>
+      </trans-unit>
+      <trans-unit id="3119050750831891643" datatype="html">
+        <source>This password is incorrect.</source>
+        <target>Ce mot de passe est incorrect</target>
+      </trans-unit>
+      <trans-unit id="5113647226865218037" datatype="html">
+        <source>This email address has no account on this website.</source>
+        <target>Cette adresse email n&apos;a pas de compte sur ce site.</target>
+      </trans-unit>
+      <trans-unit id="1988081660757251720" datatype="html">
+        <source>You must click the confirmation link that you should have received by email.</source>
+        <target>Vous devez aller dans vos mail confirmer votre inscription en cliquant sur le lien que nous vous avons envoyé d&apos;abord.</target>
+      </trans-unit>
+      <trans-unit id="850080272338290812" datatype="html">
+        <source>No account?</source>
+        <target>Pas de compte ?</target>
+      </trans-unit>
+      <trans-unit id="8999284073934392732" datatype="html">
+        <source>Login with Google</source>
+        <target>Se connecter avec Google</target>
+      </trans-unit>
+      <trans-unit id="6005801113696805305" datatype="html">
+        <source>The replay game is loading, please wait, it should not take long.</source>
+        <target>Le partie de revanche se charge. Veuillez attendre, cela ne devrait pas prendre longtemps.</target>
+      </trans-unit>
+      <trans-unit id="5120671221766405888" datatype="html">
+        <source>Game does not exist</source>
+        <target>Partie inexistante</target>
+      </trans-unit>
+      <trans-unit id="5769704000858519890" datatype="html">
+        <source>The game you tried to join does not exist anymore.</source>
+        <target>La partie que vous avez essayé de rejoindre n&apos;existe plus.</target>
+      </trans-unit>
+      <trans-unit id="2009811124619716606" datatype="html">
+        <source>Create an online game</source>
+        <target>Créer une partie en ligne</target>
+      </trans-unit>
+      <trans-unit id="5561648955936795459" datatype="html">
+        <source>Use simple mechanics to push 6 of the opponent&apos;s pieces out of the board!</source>
+        <target>Utilisez des mécaniques simples pour pousser 6 pièces adverses hors du plateau!</target>
+      </trans-unit>
+      <trans-unit id="1626489690215042302" datatype="html">
+        <source>The international version of the most famous African strategy game!</source>
+        <target>La version internationale du jeu africain de stratégie le plus connu !</target>
+      </trans-unit>
+      <trans-unit id="1207528295664437538" datatype="html">
```

Comment:
Sinon je sais pas trop quoi penser des superlatifs là, j'en avais mis initialement ?

Reply:
c'est effectivement pas ouf de superlativer je trouve, c'est le texte initial oui il me semble

Reply:
The internationa version of the famous African strategy game!
et
La version internationale du fameux jeu de stratégie africain[insécablu]!

S.i.p